src/lib/zAuction.ts

```diff
@@ -11,6 +11,16 @@ export interface NftIdBidsDto {
 	date: string;
 }
 
+export interface NftIdBidsOfListsDto {
```

Comment:
This is a bit of a confusing interface name; perhaps you could make it more simple by calling it `NftBidDto`